third_party/FMI/eplusModelFunctions.h

```diff
@@ -46,48 +47,46 @@
 EnergyPlus Functions for FMI for Co-Simulation
 
 ****************************************************/
-DllExport fmiComponent fmiEPlusInstantiateSlave(char* fmuResFolder,
-	fmiInteger *sizefmuResFolder, fmiReal *timeOut, fmiInteger *visible, 
-	fmiInteger *interactive, fmiInteger *loggingOn, fmiInteger *index);
+DllExport fmiComponent fmiEPlusInstantiateSlave(char *fmuResFolder,
+                                                fmiInteger *sizefmuResFolder,
+                                                fmiReal *timeOut,
+                                                fmiInteger *visible,
+                                                fmiInteger *interactive,
+                                                fmiInteger *loggingOn,
+                                                fmiInteger *index);
```

Comment:
Plenty of changes are due to the fact that I clang-formatted all the files in the FMI and FMUParser folders, as they were pretty messy.